examples/firebase-auth-firestore/app/routes/index.tsx

```diff
@@ -34,32 +34,44 @@ export type ActionData = {
 export const action: ActionFunction = async ({ request }) => {
   const { uid } = await requireAuth(request);
   const form = await request.formData();
-  if (request.method === "POST") {
-    const title = form.get("title");
-    if (typeof title !== "string")
-      return json<ActionData>({ error: "title is required" }, { status: 400 });
+  const action = form.get("action");
+  switch (action) {
+    case "create": {
+      const title = form.get("title");
+      if (typeof title !== "string" || !title.length) {
+        return json<ActionData>(
+          { error: "title is required" },
+          { status: 400 }
+        );
+      }
 
-    await addTodo(uid, title);
-    return redirect("/");
+      await addTodo(uid, title);
+      return redirect("/");
+    }
+    case "delete": {
+      const id = form.get("id");
+      if (typeof id !== "string") {
+        return json<ActionData>({ error: "id is required" }, { status: 400 });
+      }
+      await removeTodo(uid, id);
+      return redirect("/");
+    }
+    default: {
+      return json<ActionData>({ error: "unknown method" }, { status: 400 });
```

Comment:
Since we only have `create` or `delete`, we can keep the 2 `if`-statements like they were
Only need to change them to check for the `action`.